pkg/es/config/flagutils.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import "fmt"
+
+type IndexType string
+
+const (
+	SPAN         IndexType = "span"
+	SERVICE      IndexType = "service"
+	DEPENDENCIES IndexType = "dependencies"
+	SAMPLING     IndexType = "sampling"
+
+	numShards   = "num-shards-"
+	numReplicas = "num-replicas-"
+	priority    = "priority-%s-template"
+)
+
+func GetNumShardSpanFlag() string {
```

Comment:
You could drop Get, it's not idiomatic for Go
```suggestion
func NumShardSpanFlag() string {
```